### ISSUE TITLE
Arabic characters stripped entirely

### ISSUE THREAD

**Issue by Porges** [withoutboats/heck#41]
Found in https://github.com/tailhook/knuffel/issues/9:

```rs
assert_eq!("الطاب", "الطاب".to_kebab_case());
```

Results in:

```
thread 'main' panicked at 'assertion failed: `(left == right)`
  left: `"الطاب"`,
 right: `""`', src\main.rs:4:5
```

**Comment by Porges** [comment]
This does work correctly with `features = ["unicode"]`, but should the default not be to not strip these; even without the Unicode segmentation algorithm they can still be detected as "letters"?

**Comment by jplatte** [comment]
I don't really have time to maintain this crate anymore (I took over after withoutboats stopped maintaining it). I suggest looking for alternatives if what is there now doesn't suit your needs.

**Comment by jplatte** [comment]
Fixed by #41. I will publish 0.5.0-rc.1 in the next days (or weeks, worst case).
